src/base/preferences.cpp

```diff
@@ -1197,6 +1197,26 @@ void Preferences::setTrayIconStyle(const TrayIcon::Style style)
 }
 #endif
 
+Utils::Misc::unitType Preferences::transferSpeedUnitsPrefix() const
+{
+    return Utils::Misc::unitType(value("Preferences/Advanced/TransferSpeedUnitsPrefix", static_cast<int>(Utils::Misc::unitType::decimalPrefixBits)).toInt());
```

Comment:
IMO this option belongs somewhere in the "Speed" page, not advanced.

Reply:
>The default settings are bits with decimal prefix for transfer speeds and bytes with decimal prefix for file sizes.

👎 
I don't think using decimal prefixes is a good default option.

Reply:
I'm following this Ubuntu [guide](https://wiki.ubuntu.com/UnitsPolicy#Correct_basis) for the default setting. I agree that using decimal prefixes doesn't make sense but it's the way users are used to.
Anyway, with this patch the user can change the prefixes. This is why I think it is useful.